command/command_test.go

```diff
@@ -0,0 +1,250 @@
+package command
+
+import (
+	"reflect"
+	"testing"
+)
+
+const (
```

Comment:
Can we add a test where there are newline characters in a YAML multi-line string?

```yaml
with: 
  thing: |-
    this is my string
    it has many lines

    some of them even
    have new new lines
```

Reply:
Done.